env.local-mode

```diff
@@ -0,0 +1,3 @@
+SERVER_PORT=8080
+SERVER_ADDRESS=localhost
+
```

Comment:
please remove this file 
if needed you can create  `.env.local` instead
check my additional comments on `.env.dev` file